src/Longrunning/OperationsClient.php

```diff
@@ -0,0 +1,481 @@
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+/*
+ * GENERATED CODE WARNING
+ * This file was generated from the file
+ * https://github.com/google/googleapis/blob/master/google/longrunning/operations.proto
+ * and updates to that file get reflected here through a refresh process.
+ *
+ * EXPERIMENTAL: this client library class has not yet been declared beta. This class may change
+ * more frequently than those which have been declared beta or 1.0, including changes which break
+ * backwards compatibility.
+ */
+
+namespace Google\Longrunning;
+
+use Google\GAX\AgentHeaderDescriptor;
+use Google\GAX\ApiCallable;
+use Google\GAX\CallSettings;
+use Google\GAX\GrpcConstants;
+use Google\GAX\GrpcCredentialsHelper;
+use Google\GAX\PageStreamingDescriptor;
+use google\longrunning\CancelOperationRequest;
+use google\longrunning\DeleteOperationRequest;
+use google\longrunning\GetOperationRequest;
+use google\longrunning\ListOperationsRequest;
+use google\longrunning\OperationsGrpcClient;
+
+/**
+ * Service Description: Manages long-running operations with an API service.
+ *
+ * When an API method normally takes long time to complete, it can be designed
+ * to return [Operation][google.longrunning.Operation] to the client, and the client can use this
+ * interface to receive the real response asynchronously by polling the
+ * operation resource, or pass the operation resource to another API (such as
+ * Google Cloud Pub/Sub API) to receive the response.  Any API service that
+ * returns long-running operations should implement the `Operations` interface
+ * so developers can have a consistent client experience.
+ *
+ * EXPERIMENTAL: this client library class has not yet been declared beta. This class may change
+ * more frequently than those which have been declared beta or 1.0, including changes which break
+ * backwards compatibility.
+ *
+ * This class provides the ability to make remote calls to the backing service through method
+ * calls that map to API methods. Sample code to get started:
+ *
+ * ```
+ * try {
+ *     $operationsClient = new OperationsClient();
+ *     $name = "";
+ *     $response = $operationsClient->getOperation($name);
+ * } finally {
+ *     if (isset($operationsClient)) {
+ *         $operationsClient->close();
+ *     }
+ * }
+ * ```
+ *
+ * Many parameters require resource names to be formatted in a particular way. To assist
+ * with these names, this class includes a format method for each type of name, and additionally
+ * a parse method to extract the individual identifiers contained within names that are
+ * returned.
+ */
+class OperationsClient
+{
+    /**
+     * The default address of the service.
+     */
+    const SERVICE_ADDRESS = 'longrunning.googleapis.com';
```

Comment:
In Toolkit, add logic to not populate SERVICE_ADDRESS and $defaultScopes in PHP. This is configured for longrunning like this:

```
  required_constructor_params:
  - service_address
  - scopes
```